### PR TITLE
Pic 2900 three

### PR DESCRIPTION
**Pull request by rschallapalli**

### CODE REVIEW

File: integration-tests/integration/common/common.spec.js [hunk 1]
```diff
@@ -541,6 +541,28 @@ And('I should see the {string} query have the value {string}', ($string, $value)
...
 
+// common pac filter panel specs
+
+When('I click the {string} filter button', $string => {
+  cy.get('.pac-filter-button').contains($string).click()
+})
+
+When('I select the {string} filter', $string => {
+  cy.get(`input[data-test="${$string}"]`).invoke('attr', 'checked', true)
+})
```
Comment (johnyfarrar) [inline integration-tests/integration/common/common.spec.js:552]: Do we use data-test or data-cy? I think I've used cy and shouldn't!

File: integration-tests/integration/features/cases-in-progress-list-filter.feature [hunk 2]
```diff
@@ -108,19 +63,17 @@ Feature: Cases In progress List filters
...
+    And I select the "Other" filter
     And I click the "Outcome type" filter button
     And I click the "Apply filters" button
 
     And I should see the "outcomeType" query have the value "OTHER"
 
-    Then I should see a tab with text "In progress (0)"
-
-    And I should see a count of "0 cases"
+#    Then I should see a tab with text "In progress (0)"
```
Comment (johnyfarrar) [inline integration-tests/integration/features/cases-in-progress-list-filter.feature:72]: Is this going to be put back when we do the dynamic counts?

File: server/routes/handlers/outcomes/getCasesInProgressHandler.js [hunk 3]
```diff
@@ -7,19 +9,25 @@ const getCasesInProgressHandler = caseService => async (req, res) => {
     session
   } = req
 
-  const filters = getOutcomeTypesListFilters(req.query)
-  const filtersApplied = filters.map(filterObj => filterObj.items.filter(item => item.checked).length).pop()
+  const response = await caseService.getOutcomesList(courtCode, req.query, sorts, state)
+
+  const cases = response.cases
```
Comment (johnyfarrar) [inline server/routes/handlers/outcomes/getCasesInProgressHandler.js:14]: Could use destructuring like line 6 but I'm not going to reject a PR for that

File: server/routes/handlers/outcomes/getCasesInProgressHandler.js [hunk 4]
```diff
@@ -7,19 +9,25 @@ const getCasesInProgressHandler = caseService => async (req, res) => {
     session
   } = req
 
-  const filters = getOutcomeTypesListFilters(req.query)
-  const filtersApplied = filters.map(filterObj => filterObj.items.filter(item => item.checked).length).pop()
+  const response = await caseService.getOutcomesList(courtCode, req.query, sorts, state)
```
Comment (johnyfarrar) [inline server/routes/handlers/outcomes/getCasesInProgressHandler.js:12]: `req.query` is used a couple of times - should we make it a variable on line 6?

File: server/routes/handlers/outcomes/getCasesToResultHandler.js [hunk 5]
```diff
@@ -1,20 +1,21 @@
...
     params: { courtCode, title, sorts, state },
     params
   } = req
 
-  const filters = getOutcomeTypesListFilters(req.query)
-
-  const filtersApplied = filters.map(filterObj => filterObj.items.filter(item => item.checked).length).pop()
-
-  const response = await caseService.getOutcomesList(courtCode, filters, sorts, state)
+  const response = await caseService.getOutcomesList(courtCode, req.query, sorts, state)
```
Comment (johnyfarrar) [inline server/routes/handlers/outcomes/getCasesToResultHandler.js:9]: Again the same with the query.... I feel like I'm being overly picky though so you can ignore...

**Review by johnyfarrar — COMMENTED** [review]
I've left some comments but nothing I'm going to cry about

File: integration-tests/integration/features/cases-in-progress-list-filter.feature (hunk 2 above)
Comment (rschallapalli) [reply integration-tests/integration/features/cases-in-progress-list-filter.feature:72]: Yes

File: integration-tests/integration/common/common.spec.js (hunk 1 above)
Comment (rschallapalli) [reply integration-tests/integration/common/common.spec.js:552]: we have both in the code base. what is the preferred one?

File: integration-tests/integration/features/cases-in-progress-list-filter.feature (hunk 2 above)
Comment (rschallapalli) [reply integration-tests/integration/features/cases-in-progress-list-filter.feature:72]: And I should see a count of "0 cases" - will not be back as this is not as part of the design/AC

File: server/routes/handlers/outcomes/getCasesInProgressHandler.js (hunk 3 above)
Comment (rschallapalli) [reply server/routes/handlers/outcomes/getCasesInProgressHandler.js:15]: done

File: server/routes/handlers/outcomes/getCasesInProgressHandler.js (hunk 4 above)
Comment (rschallapalli) [reply server/routes/handlers/outcomes/getCasesInProgressHandler.js]: done

File: server/routes/handlers/outcomes/getCasesToResultHandler.js (hunk 5 above)
Comment (rschallapalli) [reply server/routes/handlers/outcomes/getCasesToResultHandler.js]: done

File: integration-tests/integration/common/common.spec.js (hunk 1 above)
Comment (johnyfarrar) [reply integration-tests/integration/common/common.spec.js:552]: I don't actually mind at all

**Review by johnyfarrar — APPROVED** [review]
Great work! Let's add a ticket to tidy the naming of the data-test things!

**Pull request merged**